components/02-elements/tab/tab.js

```diff
@@ -3,38 +3,31 @@
 (function() { // eslint-disable-line
   const tab                = [...document.querySelectorAll('.tab')],
         activeTitleClass   = 'tab__title--active',
-        activeContentClass = 'tab__content--active',
-        mediaQuery         = window.matchMedia('(min-width: 768px)');
+        activeContentClass = 'tab__content--active';
 
   tab.forEach(element => {
-    const title   = element.querySelectorAll('.tab__title'),
-          content = element.querySelectorAll('.tab__content');
+    const children = Array.from(element.children);
 
-    title.forEach(key => key.addEventListener('click', event => {
-      const self = event.currentTarget,
-            tabId = self.dataset.tab;
+    children.forEach(item => {
 
-      if (!self.classList.contains(activeTitleClass)) {
-        title.forEach(key => key.classList.remove(activeTitleClass));
-        self.classList.add(activeTitleClass);
-      }
+      if (item.classList.contains('tab__title')) {
+        item.addEventListener('click', (e) => {
+          const self = e.currentTarget,
```

Comment:
It's not necessary, you are listening events on one element `item`

Reply:
Changed.